frontend/src/components/maps/leaflet/LayerPopup/hooks/useLayerQuery.tsx

```diff
@@ -78,10 +80,40 @@ export const handleParcelDataLayerResponse = (
     .then((resp: FeatureCollection<Geometry, GeoJsonProperties>) => {
       saveParcelDataLayerResponse(resp, dispatch, latLng);
     })
-    .catch((axiosError: any) => {
+    .catch((axiosError: AxiosError) => {
       dispatch(error(parcelLayerDataSlice.reducer.name, axiosError?.response?.status, axiosError));
     });
 };
+let loadingToastId: React.ReactText | undefined = undefined;
+let errorToastId: React.ReactText | undefined = undefined;
+const MAX_RETRIES = 2;
+const wfsAxios = () => {
+  const instance = axios.create({ timeout: 5000 });
+  instance.defaults.raxConfig = {
+    retry: MAX_RETRIES,
+    instance: instance,
+    shouldRetry: (error: AxiosError) => {
+      const cfg = rax.getConfig(error);
+      if (
+        (!errorToastId || !toast.isActive(errorToastId)) &&
+        cfg?.currentRetryAttempt === MAX_RETRIES
+      ) {
+        loadingToastId && toast.dismiss(loadingToastId);
+        errorToastId = toast.error(MAP_UNAVAILABLE_STR, { autoClose: 10000 });
```

Comment:
If we have made 3 requests for the same data and none of them have succeeded, it seems fair to say that data bc may be down or having performance issues.